index.html

```diff
@@ -172,66 +169,57 @@ <h1>Welcome to Malcolm’s CSD120 Website!</h1>
         <br /><br />
 
         <h2>My Favourite Things</h2>
-        <div class="favorite">
-          <fieldset>
-            <input
-              type="radio"
-              id="fav1"
-              name="favthing"
-              value="Sound Design"
-            />
-            <label for="fav1">Sound Design</label>
-
-            <input type="radio" id="fav2" name="favthing" value="Mixing" />
-            <label for="fav2">Mixing</label>
-
-            <input type="radio" id="fav3" name="favthing" value="Gardening" />
-            <label for="fav3">Gardening</label>
-          </fieldset>
-        </div>
-        <h3>Haiku - Pot Holes</h3>
-        <p>
-          Meandering thoughts
-          <br />
-          suddenly see pot holes turn
-          <br />
-          into the lake<br />
-          <cite>by <span class="author">Malcolm L.</span></cite>
-        </p>
+        <div class="favourites">
+            
+            <input type="radio" id="fav1" name="favthing" value="Sound Design" checked />
+            <label for="fav1"><h3>Sound Design</h3></label>
+            <div>
+            <figure>
+              <img
+                src="Images/reasonsetup.jpg"
+                alt="Studio Setup"
+                width="500"
+                height="240"
+              />
+              <figcaption>Fig.1 - Modern Studio Setup.</figcaption>
+            </figure>
+            </div>
```

Comment:
Be careful when reading instructions and don't be shy to ask for clarification if they are not clear to you!